src/base/NekRSProblem.C

```diff
@@ -247,6 +246,14 @@ NekRSProblem::initialSetup()
                  " in the [MESH] block.");
   }
 
+  if (!boundary && nekrs::hasElasticitySolver())
+    mooseError("Your nekRS .par file has 'solver = elasticity' in the [MESH] block. This solver uses\n"
+               "displacement values at a boundary of interest to calcualte the mesh velocity. This\n"
+               "mesh velocity is applied within nekRS on the same boundary to solve for fluid flow\n"
+               "in a moving mesh domain using the elasticity solver. Please indicate which boundary\n"
+               "is moving in your Cardinal sub-app input file's [Mesh] block, regardless of whether\n"
```

Comment:
I think `boundary` should be the union of all boundaries which either have a heat flux BC from MOOSE, a displacement from MOOSE, or both. We just need `boundary` to set up some internal data structures for those transfers. Can you rephrase this to reflect that all the entries in `boundary` don't _need_ to be moving? Or if there is something I'm missing here, let me know.

Reply:
Thanks for pointing that out. Yes, it should be a union of all those boundaries. How is this?
```
    mooseError("Your nekRS .par file has 'solver = elasticity' in the [MESH] block. This solver uses\n"
               "displacement values at a boundary of interest to calcualte the mesh velocity. This\n"
               "mesh velocity is applied within nekRS on the same boundary to solve for fluid flow\n"
               "in a moving mesh domain using the elasticity solver. If you intend to use the elasticity\n"
               "solver, please indicate at least one boundary of interest in your Cardinal sub-app\n"
               "input file's [Mesh] block, regardless of whether you are using a volume mirror or not.");
```

Reply:
looks great, I'd perhaps put `boundary` within single quotes in the second-to-last line, like `'boundary'`, so that it's clear exactly what the user should do to address this error.

Reply:
This change has been made, thanks!